app/code/Magento/Sales/Test/Mftf/Test/StorefrontPlaceOrderAndCheckAddressesAreDifferentOnAdminOrderView.xml

```diff
@@ -0,0 +1,98 @@
+        <click selector="{{CheckoutShippingSection.next}}" stepKey="clickNext1"/>
+
+        <!-- Place order -->
+        <waitForElement selector="{{CheckoutPaymentSection.placeOrder}}" time="30" stepKey="waitForPlaceOrderButtonVisible"/>
+        <click selector="{{CheckoutPaymentSection.placeOrder}}" stepKey="placeOrder"/>
+        <waitForPageLoad stepKey="waitUntilOrderPlaced"/>
+
+        <!-- Grab order number -->
+        <seeElement selector="{{CheckoutSuccessMainSection.success}}" stepKey="orderIsSuccessfullyPlaced"/>
+        <grabTextFrom selector="{{CheckoutSuccessMainSection.orderNumber22}}" stepKey="getOrderNumber"/>
+        <assertNotEmpty stepKey="assertOrderIdIsNotEmpty" after="getOrderNumber">
+            <actualResult type="const">$getOrderNumber</actualResult>
+        </assertNotEmpty>
+
+        <!-- Go to admin order view page -->
+        <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+        <actionGroup ref="AdminOrdersPageOpenActionGroup" stepKey="goToOrders"/>
+        <actionGroup ref="OpenOrderByIdActionGroup" stepKey="filterOrderGridById">
+            <argument name="orderId" value="$getOrderNumber"/>
+        </actionGroup>
+
+        <!-- Assert order addresses -->
+        <see selector="{{AdminShipmentAddressInformationSection.shippingAddress}}" userInput="{{US_Address_NY_Default_Shipping.street[0]}}" stepKey="seeShippingAddressStreet"/>
+        <see selector="{{AdminShipmentAddressInformationSection.shippingAddress}}" userInput="{{US_Address_NY_Default_Shipping.city}}" stepKey="seeShippingAddressCity"/>
+        <see selector="{{AdminShipmentAddressInformationSection.shippingAddress}}" userInput="{{US_Address_NY_Default_Shipping.postcode}}" stepKey="seeShippingAddressPostcode"/>
+        <see selector="{{AdminShipmentAddressInformationSection.shippingAddress}}" userInput="{{US_Address_NY_Default_Shipping.state}}" stepKey="seeShippingAddressState"/>
+        <see selector="{{AdminShipmentAddressInformationSection.shippingAddress}}" userInput="{{US_Address_NY_Default_Shipping.telephone}}" stepKey="seeShippingAddressTelephone"/>
+
+        <see selector="{{AdminShipmentAddressInformationSection.billingAddress}}" userInput="{{UK_With_State_Default_Billing.street[0]}}" stepKey="seeBillingAddressStreet"/>
+        <see selector="{{AdminShipmentAddressInformationSection.billingAddress}}" userInput="{{UK_With_State_Default_Billing.city}}" stepKey="seeBillingAddressCity"/>
+        <see selector="{{AdminShipmentAddressInformationSection.billingAddress}}" userInput="{{UK_With_State_Default_Billing.postcode}}" stepKey="seeBillingAddressPostcode"/>
+        <see selector="{{AdminShipmentAddressInformationSection.billingAddress}}" userInput="{{UK_With_State_Default_Billing.state}}" stepKey="seeBillingAddressState"/>
+        <see selector="{{AdminShipmentAddressInformationSection.billingAddress}}" userInput="{{UK_With_State_Default_Billing.telephone}}" stepKey="seeBillingAddressTelephone"/>
```

Comment:
Maybe we can use `VerifyBasicOrderInformationActionGroup` to do such assertions?